x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -356,7 +352,7 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
       cy.get(EMPTY_ALERT_TABLE).should('be.visible');
     });
   });
-  it('Number fields are not visible in field edit panel', () => {
+  it('should not show number fields are not visible in field edit panel', () => {
```

Comment:
should we extract `cy.get(FILTER_GROUP_CONTROL_ACTION_EDIT(idx)).click();` into a `task` function?

Reply:
also `cy.get(FILTER_FIELD_TYPE).click();`?

Reply:
Okay I extracted below statements into a function as `editSingleFilterControl` here

```javascript
    cy.get(CONTROL_FRAME_TITLE).eq(idx).realHover();
    cy.get(FILTER_GROUP_CONTROL_ACTION_EDIT(idx)).click();
```
> also cy.get(FILTER_FIELD_TYPE).click();?

I do not see any value in converting single liners like this to task, unless they are used everywhere.

Reply:
same response I gave above, while I agree, I'm extracting every click event to a task for consistency...